js/markDone.js

```diff
@@ -0,0 +1,30 @@
+import { EDIT, COMPLETE } from "/js/constants.js";
+import calculateDuration from "/js/utility/mDDuration.js";
+import createDurationElement from "/js/utility/mDDuElement.js";
+
+export default function markTodoAsCompleted(element, taskList, id) {
+  const index = taskList.findIndex((item) => item.id === id);
+  if (index === -1) {
+    return;
+  }
+  taskList[index].done = true;
```

Comment:
task could be updated directly avoiding the index operator.

```suggestion
  const task = taskList.find((item) => item.id === id);
  if (!task) {
    return;
  }
  
  task.done = true;
```

Reply:
done!